src/pages/all-about-carbon/who-uses-carbon.mdx

```diff
@@ -63,12 +63,11 @@ Here are some ways designers can begin engaging with Carbon.
 
 #### Get the tools
 
-- Visit [Designing > Getting Started](/designing/get-started/) on the Carbon
-  website and install the design kits.
+- Visit the [Getting Started](/designing/get-started/) page on the Carbon
```

Comment:
```suggestion
- Visit the [Get started](/designing/get-started/) page on the Carbon
```

Just a small edit to match the name of the actual page it is linked to for consistency.